http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/AKSInterpreter.scala

```diff
@@ -0,0 +1,249 @@
+          config.terraAppSetupChartConfig.chartVersion,
+          org.broadinstitute.dsp.Values(
+            s"cloud=azure,serviceAccount.annotations.azureManagedIdentityClientId=${uami.clientId},serviceAccount.name=${ksaName.value}"
+          ),
+          true
+        )
+        .run(authContext)
+
+      // Create federated identity
+      _ <- setUpFederatedIdentity(params.cloudContext,
+                                  landingZoneResources.clusterName,
+                                  params.managedIdentityName,
+                                  namespaceName,
+                                  ksaName
+      )
+
+      // Create relay hybrid connection pool
+      hcName = RelayHybridConnectionName(params.appName.value)
+      primaryKey <- azureRelayService.createRelayHybridConnection(landingZoneResources.relayNamespace,
+                                                                  hcName,
+                                                                  params.cloudContext
+      )
+
+      // Build values and install chart
+      values = buildCromwellChartOverrideValues(app.release,
+                                                params.cloudContext,
+                                                ksaName,
+                                                app.samResourceId,
+                                                landingZoneResources,
+                                                hcName,
+                                                primaryKey
+      )
+      _ <- helmClient
+        .installChart(
+          app.release,
+          app.chart.name,
+          app.chart.version,
+          values,
+          createNamespace = true
+        )
+        .run(authContext)
+
+      _ <- logger.info(ctx.loggingCtx)(
+        s"Finished app creation for app ${params.appName.value} in cluster ${landingZoneResources.clusterName.value} in cloud context ${params.cloudContext.asString}"
+      )
+
+      // TODO: poll app!!
+
+    } yield ()
+
+  private[util] def buildCromwellChartOverrideValues(release: Release,
+                                                     cloudContext: AzureCloudContext,
+                                                     ksaName: ServiceAccountName,
+                                                     samResourceId: SamResourceId,
+                                                     landingZoneResources: LandingZoneResources,
+                                                     relayHcName: RelayHybridConnectionName,
+                                                     relayPrimaryKey: PrimaryKey
+  ): Values =
+    Values(
+      List(
+        // azure resources configs
+        raw"config.resourceGroup=${cloudContext.managedResourceGroupName.value}",
+        // TODO: missing, what are these?
+//      raw"config.azureServicesAuthConnectionString=???",
+//      raw"config.applicationInsightsAccountName=???",
+//      raw"config.cosmosDbAccountName=???",
```

Comment:
cosmodDbAccountName won't be needed for Terra (we're switching TES to Postgres). The other two are related to Application Insights, which TES uses for logging. I think we've been talking about needing to set that up somewhere. In the end I think we'll be passing a connection string in here for it.